docs/sources/dashboards/use-dashboards.md

```diff
@@ -19,40 +19,36 @@ keywords:
-
-{{< figure src="/static/img/docs/v50/top_nav_annotated.png" width="450px" >}}
-
-- **Dashboard title** (2): This also opens the dashboard search when clicked.
-- **Add panel** (3): Use this option to add a new panel or row to the current dashboard.
-- **Star dashboard** (4): Use this option to star (or unstar) the current dashboard. Starred dashboards show up on your own home dashboard and in the navigation bar by default. It is a convenient way to mark Dashboards that you're interested in.
-- **Share dashboard** (5): Use this option to share the current dashboard by link or snapshot. You can also export the dashboard definition from the share modal.
-- **Save dashboard** (6): Use this option to save the current dashboard using its current name.
-- **Settings** (7): Use this option to open dashboard settings. Here you change dashboard name, folder, tags as well as manage variables and annotation queries.
-
-## Dashboard panels
-
-The main building block of dashboard is the panel. You add new panels via the `Add panel` button at the top of the dashboard. This will add view at the top where you can choose if you want to add a new panel, row or library panel. Library panels are panels that can be shared (reused) in many dashboards. You can move panels around by just dragging the panel header. To resize panel use the lower right corner.
-
-## Dashboard rows
-
-A dashboard row is a logical divider within a dashboard. It is used to group panels together. Rows can be collapsed or expanded allowing you to hide parts of the dashboard. Panels inside a collapsed row will not issue any queries.
-
-Use the [repeating rows]({{< relref "../panels/add-panels-dynamically/configure-repeating-rows" >}}) to dynamically create rows based on a template variable.
+This topic provides an overview of dashboard features and shortcuts, and describes how to use dashboard search. 
+
+## Dashboard feature overview
+
+The dashboard user interface provides a number of features that you can use to customize the presentation of your data. 
+
+The following image and descriptions highlights all dashboards features.
+
+{{< figure src="/static/img/docs/v91/dashboard-features/dashboard-features.png" width="700px" >}}
+
+- **Grafana home** (1): Click the Grafana home icon to XXX.
+- **Dashboard title** (2): When you click the dashboard title you can search for dashboard contained in the current folder.
+- **Share dashboard** (3): Use this option to share the current dashboard by link or snapshot. You can also export the dashboard definition from the share modal.
+- **Add panel** (4): Use this option to add a panel, dashboard row, or library panel to the current dashboard.
+- **Dashboard settings** (5): Use this option to change dashboard name, folder, and tags and manage variables and annotation queries.
+- **Time picker dropdown** (6): Click to select relative time range options and set custom absolute time ranges.
+- **Zoom out time range** (7): xxx
```

Comment:
@axelavargas , what does zoom out time range do? This was not documented in the previous docs.